content/en/docs/concepts/storage/volumes.md

```diff
@@ -401,6 +445,17 @@ spec:
     fsType: ext4
 ```
 
+#### CSI Migration
+
+{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+
+The CSI Migration feature for GCE PD, when enabled, shims all plugin operations
+from the existing in-tree plugin to the `pd.csi.storage.gke.io` Container
```

Comment:
The other csi drivers don't actually specify the full plugin name. Which way do we want to go?